addons/l10n_in_ewaybill_stock/models/l10n_in_ewaybill.py

```diff
@@ -282,7 +282,7 @@ def _check_configuration(self):
 
     def _check_transporter(self):
         error_message = []
-        if self.transporter_id and not self.transporter_id.vat:
+        if self.transporter_id and self.transporter_id.vat in (False, "/") and (self.mode != "1" or not self.vehicle_no):
```

Comment:
```suggestion
        if self.transporter_id and not self.transporter_id.vat and (self.mode != "1" or not self.vehicle_no):
```
I appreciate the point to be considered for `"/"` for no vat it's not just one case we are considering here. If we consider "/" to be empty then we should apply it every where applicable i.e. E-invoice but after detail use case study with the PO

I would prefer different task and PR for this one after confirming it with the PO